draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -153,8 +155,9 @@ For any remote host ICE candidate received by the ICE agent, the following proce
 
 4. Otherwise, ignore the candidate.
 
-Multicast DNS resolution might end up retrieving both an IPv4 and IPv6 address.
-In that case, the IPv6 address may be used preferably to the IPv4 address.
+An ICE agent may use a host-name resolver that transparently supports both Multicast and Unicast DNS. In this case the resolution of a ".local" name may happen through Unicast DNS, see {{RFC6762}} section 3. This leads to the following consideration.
+
+An ICE agent that supports mDNS candidates MUST support the situation where the host-name resolution resuts in more than one IP address. In this case The ICE agent takes exactly one of the resolved IP addresses and ignores the others. The ICE agent SHOULD, if available, use the first IPv6 address resolved, otherwise the first IPv4 address.
```

Comment:
Since these paragraphs are mostly talking about the same thing, I think they should probably be combined.

Also: host-name -> hostname, resuts -> results, "The ICE agent" -> "the ICE agent".